src/components/FDC3Tester/src/Apps.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ApiExample from './components/ApiExample'
+
+export default function Apps() {
+
+  return (
+    <div>
+      <h2>Apps</h2>
+      {/* Open */}
+      <ApiExample
+        apiName="open"
+        title="Open"
```

Comment:
The burger menu doesn't work for Open, possibly because of the capitalization here.